.github/workflows/frontend-verify.yml

```diff
@@ -33,3 +33,23 @@ jobs:
           REACT_APP_BUILD_TIMESTAMP=$(date +%s) 
           REACT_APP_DEV_HINT=true
           npm run build 
+  
+  e2e-test:
+    needs: "lint-and-compile"
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.10.0'
+      - name: Install dependencies 
+        run: |
+          cd frontend
+          npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: |
+          cd frontend
+          npm run e2e-test
```

Comment:
Does it not require a certain backend/api to be started beforehand? @jvorcak

Reply:
Yes I am working on this. Not sure what's the best way to run the right version of backend with the appropriate configs.

Reply:
@weeco Playwright should be able to do this, but I'm having some issues with running it as part of PW. @bojand can we start the frontend for this in GH actions ?